low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -129,9 +130,10 @@ def add_args(parser: str, cls: str, name: str):
     def validate(self):
         """Validate post install command arguments and given input."""
         if not self.args.config:
-            raise SetupError(errno.EINVAL,
-                "%s - Argument validation failure. %s",
+            msg = ("%s - Argument validation failure. %s",
                 self.name, "Global config URL is required.")
+            logger.error(msg)
```

Comment:
This should be,
```
 msg = "%s - Argument validation failure. %s" %(
                self.name, "Global config URL is required.")
```

Reply:
I can see this at many places. Please correct it wherever applicable.

Reply:
Addressed. Done the changes everywhere